miraclelinux/meta-debian-extended
# Fix license warnings

# Purpose of pull request

This pr fixes license warnings for fcgi and openldap.

# Test
## How to test

Build fcgi and openldap to see warning is dismissed.

build fcgi

```
$ bitbake fcgi -c cleanall ; bitbake fcgi
```

build openldap

```
$ bitbake openldap -c cleanall ; bitbake openldap
```

also build core-image-minimal

add following lines to conf/local.conf

```
IMAGE_INSTALL_append = " openldap-slapd openldap-bin"
IMAGE_INSTALL_append = " fcgi"
```

Then build.

```
$ bitbake core-image-minimal -c cleanall ; bitbake core-image-minimal
```

## Test result

build fcgi.

```
$ bitbake fcgi 
Loading cache: 100% |##################################################################################################################################################################| Time: 0:00:00
Loaded 2362 entries from dependency cache.
NOTE: Resolving any missing task queue dependencies

Build Configuration:
BB_VERSION           = "1.42.0"
BUILD_SYS            = "x86_64-linux"
NATIVELSBSTRING      = "ubuntu-18.04"
TARGET_SYS           = "aarch64-emlinux-linux"
MACHINE              = "qemuarm64"
DISTRO               = "emlinux"
DISTRO_VERSION       = "2.4"
TUNE_FEATURES        = "aarch64 armv8a crc"
TARGET_FPU           = ""
meta                 
meta-yocto-bsp       = "HEAD:d4b57c68b22027c2bedff335dee06af963e4f8a8"
meta-debian          = "HEAD:fca5f3f1dad606bfd73b733671b70a518be28f39"
meta-debian-extended = "fix-license-warnings:6edcaaf7c995063c68a487a34c7ff9db7b83e19d"
meta-emlinux         = "HEAD:26868b5c069922062b1cfd7b258cfef003e2ca4d"

Initialising tasks: 100% |#############################################################################################################################################################| Time: 0:00:00
Sstate summary: Wanted 6 Found 0 Missed 6 Current 76 (0% match, 92% complete)
NOTE: Executing SetScene Tasks
NOTE: Executing RunQueue Tasks
NOTE: Tasks Summary: Attempted 601 tasks of which 585 didn't need to be rerun and all succeeded.

```

build openldap

```
$ bitbake openldap 
Loading cache: 100% |##################################################################################################################################################################| Time: 0:00:00
Loaded 2362 entries from dependency cache.
NOTE: Resolving any missing task queue dependencies

Build Configuration:
BB_VERSION           = "1.42.0"
BUILD_SYS            = "x86_64-linux"
NATIVELSBSTRING      = "ubuntu-18.04"
TARGET_SYS           = "aarch64-emlinux-linux"
MACHINE              = "qemuarm64"
DISTRO               = "emlinux"
DISTRO_VERSION       = "2.4"
TUNE_FEATURES        = "aarch64 armv8a crc"
TARGET_FPU           = ""
meta                 
meta-yocto-bsp       = "HEAD:d4b57c68b22027c2bedff335dee06af963e4f8a8"
meta-debian          = "HEAD:fca5f3f1dad606bfd73b733671b70a518be28f39"
meta-debian-extended = "fix-license-warnings:6edcaaf7c995063c68a487a34c7ff9db7b83e19d"
meta-emlinux         = "HEAD:26868b5c069922062b1cfd7b258cfef003e2ca4d"

Initialising tasks: 100% |#############################################################################################################################################################| Time: 0:00:00
Sstate summary: Wanted 8 Found 1 Missed 7 Current 455 (12% match, 98% complete)
NOTE: Executing SetScene Tasks
NOTE: Executing RunQueue Tasks
NOTE: Tasks Summary: Attempted 2102 tasks of which 2086 didn't need to be rerun and all succeeded.
```

build core-image-minimal

```
$ bitbake core-image-minimal
Loading cache: 100% |##################################################################################################################################################################| Time: 0:00:00
Loaded 2362 entries from dependency cache.
NOTE: Resolving any missing task queue dependencies

Build Configuration:
BB_VERSION           = "1.42.0"
BUILD_SYS            = "x86_64-linux"
NATIVELSBSTRING      = "ubuntu-18.04"
TARGET_SYS           = "aarch64-emlinux-linux"
MACHINE              = "qemuarm64"
DISTRO               = "emlinux"
DISTRO_VERSION       = "2.4"
TUNE_FEATURES        = "aarch64 armv8a crc"
TARGET_FPU           = ""
meta                 
meta-yocto-bsp       = "HEAD:d4b57c68b22027c2bedff335dee06af963e4f8a8"
meta-debian          = "HEAD:fca5f3f1dad606bfd73b733671b70a518be28f39"
meta-debian-extended = "fix-license-warnings:6edcaaf7c995063c68a487a34c7ff9db7b83e19d"
meta-emlinux         = "HEAD:26868b5c069922062b1cfd7b258cfef003e2ca4d"

WARNING: /home/masami/emlinux/truly-latest/build/../repos/meta-debian/recipes-kernel/linux/linux-base_git.bb.do_build is tainted from a forced run                                     | ETA:  0:00:00
Initialising tasks: 100% |#############################################################################################################################################################| Time: 0:00:01
Sstate summary: Wanted 5 Found 1 Missed 4 Current 913 (20% match, 99% complete)
NOTE: Executing SetScene Tasks
NOTE: Executing RunQueue Tasks
NOTE: Tasks Summary: Attempted 3185 tasks of which 3171 didn't need to be rerun and all succeeded.

Summary: There was 1 WARNING message shown.
```
